doc/en/object/general.md

```diff
@@ -63,21 +63,21 @@ operator; setting the property to `undefined` or `null` only removes the
 *value* associated with the property, but not the *key*.
 
     var obj = {
```

Comment:
I'm less keen on this change. If we're not replacing 'foo' with something meaningful & realisitic, I think foo, bar, baz is actually preferable to 'a','b','c'. More people will realise it's an explicit 'name does not matter' placeholder.

Reply:
Oh, that was the exact reason in my head to make it abc.

Because of this comment: https://github.com/BonsaiDen/JavaScript-Garden/issues/261#issuecomment-106865800
> foo and bar are specially confusing for non english speakers (like me) because we can't translate/understand these words

foo, bar, etc. might seem like they're supposed to make sense. abc are alphabets as placeholder 😅